tests/queue_functions/queue_trigger_untyped/main.py

```diff
@@ -0,0 +1,8 @@
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def main(msg: str) -> str:
```

Comment:
please add another test without annotation.  The purpose of explicity specifying dataType in function.json is to to binding to basic types without annotations

Reply:
I believe this should no longer be applicable since we are removing the dependency on data type?

Reply:
We do need to have tests for both i.e dataType is specified and not specified.